src/main/java/cn/nukkit/block/BlockLight.java

```diff
@@ -0,0 +1,63 @@
+package cn.nukkit.block;
+
+import cn.nukkit.item.Item;
+import cn.nukkit.math.AxisAlignedBB;
+import cn.nukkit.utils.Identifier;
+
+public class BlockLight extends BlockTransparent {
+
+    public BlockLight(Identifier identifier) {
+        super(identifier);
+    }
+
+    @Override
+    public void setDamage(int meta) {
+        super.setDamage(meta & 0xF);
+    }
+
+    @Override
+    public int getLightLevel() {
+        return getDamage() & 0xF;
+    }
+
+    @Override
+    public AxisAlignedBB getBoundingBox() {
+        return null;
+    }
+
+    @Override
+    public boolean canWaterlogSource() {
```

Comment:
The block in the master branch had waterlogging level = 2 so the migrated block needs to set both source and flowing properties to true

Reply:
Ops I forgot this. I'll fix it

Reply:
Fixed at c64b4fe52aa0491e1ac83656be94eab0cd1e4d46